repos/dir.props

```
@@ -137,6 +137,8 @@
     <ArcadeSdkOverride Include="Microsoft.DotNet.Arcade.Sdk" Group="ARCADE" Version="$(arcadeOutputPackageVersion)"/>
     <ArcadeBootstrapSdkOverride Include="Microsoft.DotNet.Arcade.Sdk" Group="ARCADE" Version="$(ArcadeBootstrapVersion)" Location="$(ArcadeBootstrapDir)microsoft.dotnet.arcade.sdk/$(ArcadeBootstrapVersion)" />
     <ArcadePackagingOverride Include="Microsoft.DotNet.Build.Tasks.Packaging" Group="ARCADE_PACKAGING" Version="$(arcadeOutputPackageVersion)"/>
+    <ArcadeTargetFrameworkSdkOverride Include="Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk" Group="ARCADE_TGT_FWK_SDK" Version="$(arcadeOutputPackageVersion)"/>
+    <ArcadeSharedFrameworkSdkOverride Include="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" Group="ARCADE_SHARED_FWK_SDK" Version="$(arcadeOutputPackageVersion)"/>
```

(Nit) The usual abbr for framework is fx:

```suggestion
    <ArcadeTargetFrameworkSdkOverride Include="Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk" Group="ARCADE_TGT_FX_SDK" Version="$(arcadeOutputPackageVersion)"/>
    <ArcadeSharedFrameworkSdkOverride Include="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" Group="ARCADE_SHARED_FX_SDK" Version="$(arcadeOutputPackageVersion)"/>
```

---

Fixed.